static/js/containers/ProductDetailEnrollApp.js

```diff
@@ -58,7 +60,7 @@ export class ProductDetailEnrollApp extends React.Component<Props> {
               >
                 Enroll now
               </a>
-            ) : (
+            ) : run && now.isAfter(moment(run.enrollment_start)) ? (
```

Comment:
The current date should be in between enrollment start and end date

Reply:
@umarmughal824 I was trying to produce it but as per my understanding if the enrollment end_date is reached then we will get an empty run. 
Can you make sure, you are checking the `Enroll Now` button with the logged in user ?

Reply:
We have an `Enroll Now` labelled on line `61`